templates/enterprise-license-job.yaml

```diff
@@ -65,8 +65,35 @@ spec:
             {{- end}}
           command:
             - "/bin/sh"
-            - "-ec"
+            - "-c"
             - |
+              log() {
+                echo "$(date +%d-%m-%Y" "%H:%M:%S) $@"
+              }
+
+              log "Waiting for leader to be elected..."
+              while true; do
```

Comment:
It'd be good to add a timeout. It's probably not good to just have this job running forever in case the server is not working for a long period of time.